exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/metadata/Metadata.java

```diff
@@ -91,9 +95,14 @@
 public class Metadata {
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(Metadata.class);
 
-  public static final String[] OLD_METADATA_FILENAMES = {".drill.parquet_metadata.v2"};
+  public static final String[] OLD_METADATA_FILENAMES = {".drill.parquet_metadata", ".drill.parquet_metadata.v2"};
   public static final String METADATA_FILENAME = ".drill.parquet_metadata";
   public static final String METADATA_DIRECTORIES_FILENAME = ".drill.parquet_metadata_directories";
+  public static final String FILE_METADATA_FILENAME = ".drill.parquet_file_metadata.v4";
+  public static final String METADATA_SUMMARY_FILENAME = ".drill.parquet_summary_metadata.v4";
+  public static final String[] CURRENT_METADATA_FILENAMES = {METADATA_SUMMARY_FILENAME, FILE_METADATA_FILENAME};
```

Comment:
What about `METADATA_DIRECTORIES_FILENAME`? Is it current metadata fileName?

Reply:
METADATA_DIRECTORIES_FILENAME is same across all the versions. CURRENT_METADATA_FILENAMES contains the newer version cache files. In this case summary and file metadata cache file names